mapswipe-dashboard/package.json

```diff
@@ -0,0 +1,137 @@
+{
+    "name": "mapswipe-manager-dashboard",
+    "longName": "MapSwipe Manager Dashboard",
+    "version": "1.0.0",
+    "description": "Manager Dashboard for MapSwipe",
```

Comment:
this shouldn't be `manager-dashboard`. also check the other references below and the git repo url etc.